keyboards/mt/ncr80/hotswap/hotswap.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2022 Jordan Duabe
+ * Copyright 2022 peepeetee
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+
+#define LAYOUT_tkl_ansi_wkl( \
```

Comment:
Missing print screen, scroll lock and pause/break, so this is not a TKL ANSI WKL layout.
```suggestion
#define LAYOUT( \
```